src/OrchardCore.Modules/OrchardCore.Autoroute/Views/_ViewImports.cshtml

```diff
@@ -5,6 +5,7 @@
 
 @inherits OrchardCore.DisplayManagement.Razor.RazorPage<TModel>
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@addTagHelper *, OrchardCore.Contents.TagHelpers
```

Comment:
You will need to take a direct reference to `OrchardCore.Contents.TagHelpers` project (or this will fail in production)

But looks good 👍